### PR TITLE
Added Cloud SQL Server with authorized network example

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Added example for inclusion on the following page:

https://cloud.google.com/sql/docs/sqlserver/authorize-networks

```release-note:none
```

### CODE REVIEW

File: mmv1/products/cgc/terraform.yaml [hunk 1]
```diff
@@ -51,6 +51,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_sqlserver_instance_authorized_network"
+        primary_resource_id: "default"
```
Comment (shuyama1) [inline mmv1/products/cgc/terraform.yaml:56]: We will probably need to add `primary_resource_type: "google_sql_database_instance"` for snippets for handwritten resources

**Review by shuyama1 — CHANGES_REQUESTED** [review]

**Review by shuyama1 — CHANGES_REQUESTED** [review]

File: mmv1/products/cgc/terraform.yaml [hunk 2]
```diff
@@ -51,6 +51,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
...
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_sqlserver_instance_authorized_network"
+        primary_resource_type: "google_sql_database_instance"
+        primary_resource_id: "default"
+        vars:
+          myinstance: "myinstance"
+        ignore_read_extra:
+          - "deletion_protection"
```
[the remark this answers is not in the document]
Comment (betsy-lichtenberg) [reply mmv1/products/cgc/terraform.yaml:61]: Done

File: mmv1/templates/terraform/examples/sql_sqlserver_instance_authorized_network.tf.erb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    tier = "db-custom-2-7680"
+    ip_configuration {
+      authorized_networks {
+        name = "Network Name"
+        value = "192.0.2.0/24"
+        expiration_time = "3021-11-15T16:19:00.094Z"
+      }
+    }
+  }
+}
```
Comment (shuyama1) [inline mmv1/templates/terraform/examples/sql_sqlserver_instance_authorized_network.tf.erb:17]: Looks like we need to add `deletion_protection = false` to get tests passed, as the field set to true by default but we need to be able to delete the resource in our tests.

If `deletion_protection = true` needs to be shown in CGC samples, we probably can follow what have been done in this PR (https://github.com/GoogleCloudPlatform/magic-modules/pull/5706), adding `test_vars_overrides`.

Thank you so much!

**Review by shuyama1 — CHANGES_REQUESTED** [review]

File: mmv1/templates/terraform/examples/sql_sqlserver_instance_authorized_network.tf.erb (hunk 3 above)
Comment (betsy-lichtenberg) [reply mmv1/templates/terraform/examples/sql_sqlserver_instance_authorized_network.tf.erb:18]: Done

**Review by shuyama1 — APPROVED** [review]
Tests passed. LGTM! Thank you!

**Pull request merged**